components/TwoUp/index.ts

```diff
@@ -0,0 +1,94 @@
+import './styles.css';
+import { PointerTracker, Pointer } from '../../utils/trackDragging';
+
+/**
+ * A split view that the user can adjust. The first child becomes
+ * the left-hand side, and the second child becomes the right-hand side.
+ */
+export default class TwoUp extends HTMLElement {
+  private readonly _handle = document.createElement('div');
+  /**
+   * The position of the split.
```

Comment:
In pixels? Percent?

Reply:
It's pixels now, so I'll update the comment. At some point I need to come up with a better way to handle resizing, since it's currently possible to lose the handle. One way will be to use a percent value.